src/vidible-player.js

```diff
@@ -11,9 +11,14 @@
             var videoReadyTimeout = 5000;
             var queue = [];
 
-            function loadScriptByVideoId(videoId, callback) {
-                var vidibleScriptUrl = 'http://delivery.vidible.tv/jsonp/pid=55c8aae9e4b0ca68372fb553/vid=' +
-                    videoId + '/55af9dcae4b02944c03a2eee.js';
+            var kAutoplayVidiblePlayer = '55c8aae9e4b0ca68372fb553';
```

Comment:
Thought about it more. We'll leave it so for the moment. And I'll add a configuration option later that will include:
- playerIdWhenAutoPlayIsEnabled
- playerIdWhenAutoPlayIsDisabled
- vidibleId